lte/gateway/c/session_manager/AAAClient.cpp

```diff
@@ -11,24 +11,25 @@
  * limitations under the License.
  */
 
+#include "lte/gateway/c/session_manager/AAAClient.h"
+
 #include <glog/logging.h>
 #include <grpcpp/channel.h>
 #include <grpcpp/impl/codegen/status.h>
+#include <feg/gateway/services/aaa/protos/accounting.grpc.pb.h>
```

Comment:
Not looking to have you change your PR, but I'm wondering why these went to `<>` from `""`? In the include fixup script I'm working on, I intended for repo-generated protos to use quotes (#12441).

Reply:
ooo didn't realize. IWYU seems to default to <>, so I standardized on that. I don't have a preference either way as long as there is tooling around it to automate it :D